rotkehlchen/data_migrations/manager.py

```diff
@@ -41,10 +43,14 @@ def maybe_migrate_data(self) -> None:
                 )
                 self.rotki.data.db.conn.commit()
 
-    def _perform_migration(self, migration: MigrationRecord) -> None:
+    def _perform_migration(self, migration: MigrationRecord) -> bool:
+        """Performs a single data migration and returns boolean for success/failure"""
         try:
             kwargs = migration.kwargs if migration.kwargs is not None else {}
             migration.function(rotki=self.rotki, **kwargs)
         except BaseException as e:  # lgtm[py/catch-base-exception]
-            error = f'Failed to run soft migration from version {migration.version} : {str(e)}'
-            log.error(error)
```

Comment:
@yabirgb  with this user has absolutely no indicaton that something happened. In fact ... I spent about 20 mins trying to figure out why the heck no update was happening only to realize there is no feedback. Just a log entry.